src/stories/ActionBar.stories.js

```diff
@@ -81,7 +81,7 @@ const docsDecorator = () => {
 
 export default {
 	component: ActionBar,
-	title: 'ActionBar',
+	title: 'bios/ActionBar',
```

Comment:
As I can see (I can be wrong), you are adding two modifications ins this PR. Maybe you should separe them.

I do not agree with this separation. This project was supposed to be agnostic of Sysvale's internal projects.

Reply:
We are currently building a Style Guide for the "Cidade Saudável", which will compose our design system in the future and, in the course of this construction, we are defining the standards we will use. Because "Bios" and "Cidade Saudável" having different objectives, not all components can be shared between the two applications, so that to facilitate the documentation and use, we are separating the components that are platform specific.

Reply:
This does not necessarily imply that we are going to use two component libraries. It's just a way that the storybook uses to put a title on the side bar that separates the components into groups. In our case the separation by groups would occur by application.

Reply:
If a component is project specific it doesn't belong to a open library.  The component should be extensible to be compatible with the Style Guide of each project.

Reply:
We are meeting, to discuss topics like that every day at 3pm. Would you like to chat with us today about the subject?